docs/examples/notebooks/03-Integrator.ipynb

```diff
@@ -0,0 +1,212 @@
+    "\n",
+    "Define the FPGA device on which the remote FpgaPesEnsembleNetwork will run. This name corresponds with the name in your `fpga_config` file. Recall that in the `fpga_config` file, device names are identified by the square brackets (e.g., **[de1]** or **[pynq]**). The names defined in your configuration file might differ from the example below. Here, the device **de1** is being used."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "board = 'de1'  # Change this to your desired device name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 3: Create an Input for the Model\n",
+    "We will use a piecewise step function as input, so we can see the effects of recurrence."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = nengo.Network(label='Integrator')\n",
+    "# Create a piecewise step function for input\n",
+    "with model:\n",
+    "    input_node = nengo.Node(\n",
+    "        Piecewise({\n",
+    "            0: 0,\n",
+    "            0.2: 1,\n",
+    "            1: 0,\n",
+    "            2: -2,\n",
+    "            3: 0,\n",
+    "            4: 1,\n",
+    "            5: 0\n",
+    "        }))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 4: Create the FPGA Ensemble\n",
+    "\n",
+    "Create a remote FPGA neural ensemble (`FpgaPesEnsembleNetwork`) using the board defined above, 100 neurons, 1 dimensions, and with no learning rate. We will also add the recurrent connection here."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tau = 0.05  # Synaptic time constant\n",
+    "\n",
+    "with model:\n",
+    "    # Remote FPGA neural ensemble\n",
+    "    fpga_ens = FpgaPesEnsembleNetwork(\n",
+    "        board,  # The board to use (from above)\n",
+    "        n_neurons=100,  # The number of neurons to use in the ensemble\n",
+    "        dimensions=1,  # 1 dimensions\n",
+    "        learning_rate=0,  # No learning for this example\n",
+    "        recurrence=1  # Activate the recurrent connection\n",
+    "    )\n",
+    "\n",
+    "    # Setting `recurrence=1` creates a `recurrent` connection object\n",
+    "    # with the identity transform. You can set the attributes on this\n",
+    "    # recurrent connection object, such as `.function` and `.transform`\n",
+    "    fpga_ens.recurrent.synapse = tau  # `nengo.LowPass(tau)`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 5: Connect Everything Together\n",
+    "The recurrent connection is housed on the FPGA device and is created as part of the `FpgaPesEnsembleNetwork` object, so we actually only need to connect the stimulus to the enesemble now!"
```

Comment:
The recurrent connection is housed on the FPGA device and is created as part of the `FpgaPesEnsembleNetwork` object, so the only connection that needs to be made is the input stimulus to the FPGA ensemble.